apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.ts

```diff
@@ -34,4 +35,11 @@ export class AccordionShowcaseComponent {
       type: ['string'],
     },
   ];
+  events: ApiDescriptionEvent[] = [
+    {
+      name: 'toggle',
+      description: 'Emitted when the state of accordion item has changed',
+      signature: 'Promise<Boolean>',
```

Comment:
Consider correct type:
```suggestion
      signature: 'EventEmitter<boolean>',
```